lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -50,6 +52,12 @@ namespace {
   static cl::opt<bool> EnableBigQuery("souper-exhaustive-synthesis-enable-big-query",
     cl::desc("Enable big query in exhaustive synthesis (default=false)"),
     cl::init(false));
+  static cl::opt<bool> EnableDataflowPruning("souper-dataflow-pruning",
+    cl::desc("Enable pruning based on dataflow analysis (default=true)"),
```

Comment:
default=false

Reply:
you resolved this without changing the text

Reply:
you resolved this without changing the text

Reply:
I'm talking about the text on line 63

Reply:
it says "(default=true)" but that contradicts the code below which has default = false